packages/core/src/crypto/KeyType.ts

```diff
@@ -4,4 +4,7 @@ export enum KeyType {
   Bls12381g1 = 'bls12381g1',
   Bls12381g2 = 'bls12381g2',
   X25519 = 'x25519',
+  P256 = 'p256',
+  P384 = 'p384',
+  P521 = 'p521',
```

Comment:
Why are the values here different from the spec (e.g. `p256` vs `P-256`)?

Reply:
Because we started with defining key types as lowercase in AFJ, and that's what we're now doing for consistency I guess 🤷

Reply:
This is probably related to the discussion [here](https://github.com/hyperledger/aries-framework-javascript/pull/1446/files/2104c0da82db470390399c8e3777d5d1c60132ab#r1181893677)

Reply:
> Because we started with defining key types as lowercase in AFJ, and that's what we're now doing for consistency I guess 🤷

If we aim for consistency, I'd say we follow the specs 😉. That way, people who are new to AFJ but are familiar with the specs will recognize things faster.